src/pybind/mgr/cephadm/module.py

```diff
@@ -91,11 +91,11 @@ def remoto_has_connection(self: Any) -> bool:
 """
 
 # Default container images -----------------------------------------------------
-DEFAULT_IMAGE = 'docker.io/ceph/ceph'
-DEFAULT_PROMETHEUS_IMAGE = 'docker.io/prom/prometheus:v2.18.1'
-DEFAULT_NODE_EXPORTER_IMAGE = 'docker.io/prom/node-exporter:v0.18.1'
+DEFAULT_IMAGE = 'quay.io/ceph/ceph'
+DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prom/prometheus:v2.18.1'
+DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prom/node-exporter:v0.18.1'
 DEFAULT_GRAFANA_IMAGE = 'docker.io/ceph/ceph-grafana:6.7.4'
```

Comment:
On a second look, that's not the "official" grafana container image but the one built by us.

There's almost the same image on quay.io registry too but I don't know the status of it (the name is a little bit different, with -combined suffix) [1]

https://quay.io/repository/ceph/ceph-grafana?tab=tags [1]

@dmick Do you have more information about this ?

Reply:
aha, already tehre; updated grafana link.

the only one missing now is haproxy... i don't see an official image on quay for hat.

Reply:
-combined was a test of the manifest-list universal-arch tag.  The base tag on docker.io and quay.io (6.7.4, currently) is meant to be manifest list now.  That appears not to yet be the case for quay.io; looking into why.

Reply:
Dunno.  It's there now (after running the ceph-grafana-trigger jenkins job), and I've removed the -combined tag from quay.io.  If I knew how to remove tags from docker.io I would do it there too.

Reply:
@liewegas : tl;dr: make sure you use just "6.7.4" for both

Reply:
@dmick I've deleted the -combined image on docker.io. I think there's only 3 members max in the docker.io ceph org and it's already full (and you're not in the list)

Reply:
Thanks.  I asked Sage as well, but thanks for taking care of it.